ludolph/bot.py

```diff
@@ -609,14 +611,28 @@ def _room_config(self):
             logger.error('Could not configure MUC room member list. Error was: %s (condition=%s, etype=%s)',
                          e.text, e.condition, e.etype)
 
+    @staticmethod
+    def _sleekxmpp_fix_jid(jid):
+        """
+        SleekXMPP MUC plugin (xep_0045.py) has changed and return string instead of object. Backward compatible fix!
+
+        Looks like the xep_0045.py is using getStanzaValues() in handle_groupchat_presence and get_stanza_values got
+        changed in
+        https://github.com/fritzy/SleekXMPP/commit/79f3c1ac8f1aa0b099958e824dc53c17daf9849f#diff-c12f981bbed099442fe509b80649eec3
```

Comment:
The URL is too long - please use the shorter one: https://github.com/fritzy/SleekXMPP/commit/79f3c1ac8f1aa0b099958e824dc53c17daf9849f